sdk/cosmos/azure-cosmos-benchmark/ctl/linkedin/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# This script should be executed from azure-sdk-for-java/sdk/cosmos/azure-cosmos-benchmark
+#   docker build -t linkedin-ctl-workload -f ctl/linkedin/Dockerfile .
+#
+
+FROM centos:7
```

Comment:
Why we need separate docker image , is it for linux/java version ?  @kushagraThapar  fyi, if need separate docker image , then we have to make change in our ctl image uploader yml, to include this image as well

Reply:
We wanted to come as close as possible to the conditions under which our application runs, the OS, java sdk, the JVM options (which haven't been configured yet). That's the reason for the lentos and Zulu jvm steps.